src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/FundingComplete.cshtml

```diff
@@ -1,22 +1,14 @@
 ﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
 
-@if (Model.ReservedFundingToShow != null)
-{
+
     <h3 class="das-panel__heading">Apprenticeship funding secured</h3>
     <dl class="das-definition-list das-definition-list--with-separator">
         <dt class="das-definition-list__title">Legal entity:</dt>
-        <dd class="das-definition-list__definition">@Model.ReservedFundingToShowLegalEntityName</dd>
+        <dd class="das-definition-list__definition">@(Model.AccountViewModel.Name)</dd>
         <dt class="das-definition-list__title">Training course:</dt>
         <dd class="das-definition-list__definition">@Model.ReservedFundingToShow.CourseName</dd>
         <dt class="das-definition-list__title">Start and end date: </dt>
         <dd class="das-definition-list__definition">@Model.ReservedFundingToShow.StartDate.ToString("MMMM yyyy") to @Model.ReservedFundingToShow.EndDate.ToString("MMMM yyyy")</dd>
     </dl>
-}
-
-@if (Model.RecentlyAddedReservationId != null &&
-     Model.ReservedFundingToShow?.ReservationId != Model.RecentlyAddedReservationId)
-{
-    <p>We're dealing with your request for funding, please check back later.</p>
```

Comment:
Even though the copy isn't great, this is currently displayed when the user has just added a reservation (so we have a reservation id in the url), and they have reservations stored on the account, but not the reservation just added. In this case, we display an earlier reservation, and the disclaimer that we're not showing their recently added reservation. (As well as being displayed on its own, if the user has just added a reservation, and we have no reservations at all.)

It would make more sense if the copy were changed to something like...
'We're dealing with your recent request for funding, please check back later'.

Reply:
I've created a generic partial that says "We're dealing with your request, please check back later." So we can use it for this and any other panels that might be stuck waiting for events

Reply:
Your new panel is a good addition, my point was that we should still display that text when appropriate in the existing `FundingComplete` partial also.